public/index.js

```diff
@@ -0,0 +1,60 @@
+
+fetch('https://json-server.burlingtoncodeacademy.now.sh/restaurants')
```

Comment:
Was this something you maybe forgot to remove? This is fetching from the right place, but you don't do anything with the information!